core/src/main/java/net/adoptopenjdk/icedteaweb/jnlp/element/information/InformationDesc.java

```diff
@@ -234,9 +294,24 @@ public boolean isSharingAllowed() {
      * @return the associations specified in the JNLP file
      */
     public AssociationDesc[] getAssociations() {
-        List<Object> associations = getItems(ASSOCIATION_ELEMENT);
+        List<AssociationDesc> associations = (List) getItems(ASSOCIATION_ELEMENT);
```

Comment:
Null check should be added since we use it later

Reply:
association will not be null, always returns an empty list